docs/rules/index.md

```diff
@@ -2,7 +2,6 @@
 title: List of available rules
 layout: doc
 ---
-<!-- Note: No pull requests accepted for this file. See README.md in the root directory for details. -->
```

Comment:
I removed this since it's no longer true after (https://github.com/eslint/eslint/issues/6658)